ci-operator/step-registry/gather/installer/gather-installer.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+
+echo "Get the installer gather logs if it is around"
+
+dir=/tmp/installer
+
+./openshift-install gather bootstrap --dir=${dir} --key "${SSH_PRIVATE_KEY_PATH}" || true
```

Comment:
But on the other hand, I'm pretty sure you don't even need to specify it here...

Reply:
It's either that or the bootstrap/master node IP

Reply:
Ah that makes sense